falco-libs.yaml

```diff
@@ -0,0 +1,113 @@
+package:
+  name: falco-libs
+  version: 0.13.2
+  epoch: 0
+  description: Foundational components necessary to build Falco
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - gcc
+      - cmake
+      - make
+      - git
+      - bash
+      - perl
+      - linux-headers
+      - autoconf
+      - automake
+      - m4
+      - libtool
+      - elfutils-dev
+      - libelf
+      - libelf-static
+      - patch
+      - binutils
+      - c-ares-dev
+      - curl-dev
+      - grpc-dev
+      - jq-dev
+      - yaml-cpp-dev
+      - openssl-dev
+      - protobuf-dev
+      - zlib-dev
+      - clang-16-dev
+      - bpftool
+      - gtest-dev
+      - libtbb-dev
+      - json-c-dev
+      - luajit
+      - re2-dev
+      - libbpf-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/falcosecurity/libs
+      tag: ${{package.version}}
+      expected-commit: 3977c2d638455cac5d978135bd883fa5c59ac8ed
+
+subpackages:
```

Comment:
You can use `items:` here to de-duplicate same commands and simplify.

i.e., https://github.com/wolfi-dev/os/blob/c574ef529e848f82cb1b6c0c5527420f15bb74e4/boost.yaml#L19-L21 or https://github.com/wolfi-dev/os/blob/c574ef529e848f82cb1b6c0c5527420f15bb74e4/cni-plugins.yaml#L36-L39